### DISCUSSION TITLE
How to Change Default Mouse Action Sensitivity for 'Series Scroll'

### DISCUSSION THREAD

**Discussion by renangarciarosa** [nroduit/Weasis#515] [Q&A]
I need to adjust the default mouse action sensitivity for the 'series scroll' feature in preferences
Can  provide guidance on how to do this?
Thanks

**Comment by nroduit** [comment]
It is possible but not simple as long as Weasis-Manager is available, see this [post](https://groups.google.com/g/dcm4che/c/L-OnKaLXV-o/m/BaKj-V2uEAAJ).

See mouse[ default preferences](https://github.com/nroduit/Weasis/blob/master/weasis-distributions/etc/config/base.json#L499-L525)

See [Weasis preferences](https://weasis.org/en/basics/customize/preferences/)
